scripts/build-and-push-images.sh

```diff
@@ -28,9 +32,15 @@ cd $ODS_PIPELINE_DIR
 
 for image in $IMAGES; do
     if [ "${SKIP_BUILD}" != "true" ]; then
-        echo "Building image $REGISTRY/$NAMESPACE/$image ..."
-        docker build -f build/package/Dockerfile.$image -t $REGISTRY/$NAMESPACE/$image .
+        odsImage="ods-$image" 
+        echo "Building image $REGISTRY/$NAMESPACE/$odsImage..."
+        docker build \
+            --build-arg http_proxy=$http_proxy \
+            --build-arg https_proxy=$https_proxy \
+            --build-arg HTTP_PROXY=$HTTP_PROXY \
+            --build-arg HTTPS_PROXY=$HTTPS_PROXY  \
```

Comment:
Why do we use different variables for lower and uppercase? Shouldn't those be the same value, just different spelling of the variable name?